### PR TITLE
AC_Fence: reset fences breach on disable

### PR DESCRIPTION
**Pull request by khancyr**

### CODE REVIEW

File: libraries/AC_Fence/AC_Fence.cpp [hunk 1]
```diff
@@ -97,6 +97,14 @@ AC_Fence::AC_Fence(const AP_AHRS& ahrs, const AP_InertialNav& inav) :
     }
 }
 
+void AC_Fence::enable(bool true_false)
+{
+    _enabled = true_false;
+    if (!true_false) {
```
Comment (OXINARF) [inline libraries/AC_Fence/AC_Fence.cpp]: We should change the argument name, this line is ugly. Something like `enable` is much more readable.

Comment (khancyr) [inline libraries/AC_Fence/AC_Fence.cpp]: agreed done!

**Comment by OXINARF** [comment]
@khancyr What's up with that last commit? We shouldn't do that, AC_Fence is specific to Copter and Sub, there is no reason to put it in the AP_Arming library. AP_Arming is subclassed precisely for vehicles to add checks that other vehicles don't need.

**Comment by khancyr** [comment]
@oxinarf, arf sorry... I push on the wrong branch... 
Anyway, it will come for rover soon (this is the only thing missing)  ! and probably to plane too. So I need to the move to ap_arming ☺️
Pleaze forget about the last commit, I will remove it from this PR

**Comment by khancyr** [comment]
ok, rebased and remove the wrong commit.

**Comment by rmackay9** [comment]
Looks like travis is not happy with this.  something about enable shadowing something.

**Comment by khancyr** [comment]
rebased and rename the parameter to value ... otherwise enable trigger an error...

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request closed without merging**